### PR TITLE
Adding Summit + DN CfP

### PR DESCRIPTION
**Pull request by myfear**
Requested by Arun Gupta. Needs to be approved first.

### CODE REVIEW

File: devoxx4kids.html.slim [hunk 1]
```diff
@@ -46,6 +49,31 @@ hero_class: devoxx4kids
...
+.row
+    .large-24.column
+      h2 Devoxx4Kids at Red Hat Summit and DevNation
+      p Red Hat is hosting a Devoxx4Kids event that will invite technology educators and kids together on Sunday, Jun 21 in Boston, MA. 
+
+.row
+    .large-8.column
+      h4 Give a Workshop
+      p 
+       | What are some of the suggested topics that can be submitted for the workshops?
```
Comment (paulrobinson) [inline devoxx4kids.html.slim]: This sentence seems a bit out of place. Can you clarify?

**Comment by paulrobinson** [comment]
Can you report back when Arun has approved this?

**Comment by myfear** [comment]
@paulrobinson got feedback from Arun. Incorporated now. RE: your comment. Took it from Arun's post ... shrug ...

If this builds, it is ready to go live.

**Comment by myfear** [comment]
All good. @paulrobinson .. please merge. Thanks!

**Pull request merged**